cpp/daal/src/algorithms/kmeans/kmeans_init_csr_random_batch_fpt_cpu.cpp

```diff
@@ -39,7 +39,7 @@ template class BatchContainer<DAAL_FPTYPE, randomCSR, DAAL_CPU>;
 }
 namespace internal
 {
-template class KMeansInitKernel<randomCSR, DAAL_FPTYPE, DAAL_CPU>;
+template class DAAL_EXPORT KMeansInitKernel<randomCSR, DAAL_FPTYPE, DAAL_CPU>;
```

Comment:
Please make sure, that you exported just necessary kernels, it can affect the icx migration, which requires to export just used part of kernels

Reply:
done